checks/network_checks.py

```diff
@@ -117,7 +125,7 @@ def restart_pool(self):
     def check(self, instance):
         if not self.pool_started:
             self.start_pool()
-        if threading.activeCount() > 5 * self.pool_size + 5: # On Windows the agent runs on multiple threads so we need to have an offset of 5 in case the pool_size is 1
+        if threading.activeCount() > 5 * NetworkCheck._global_current_pool_size + 1:
```

Comment:
The reason was that the windows agent is not using threads to run its different components (collector dogstatsd forwarder) anymore but multiprocessing, which was the reason we had this offset in the first place. However Windows checks use wmi which is itself multithreaded so you're right we should keep this offset. Fixing that.